src/openrct2/audio/Audio.cpp

```diff
@@ -59,9 +59,9 @@ rct_vehicle_sound_params* gVehicleSoundParamsListEnd;
 // clang-format off
 static int32_t SoundVolumeAdjust[SOUND_MAXID] =
 {
-    0,      // SOUND_LIFT_1
-    0,      // SOUND_TRACK_FRICTION_1
-    0,      // SOUND_LIFT_2
+    0,      // SOUND_LIFT_CLASSIC
+    0,      // SOUND_TRACK_FRICTION_CLASSIC_WOOD
+    0,      // SOUND_FRICTION_CLASSIC
```

Comment:
I suppose you mean friction wheels? That's indeed one usage (the Junior RC, for example), but it's also used by rides like the dinghy slide, which uses a conveyor belt.